src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -10,24 +10,27 @@
                  || healthData.rgw != null
                  || healthData.fs_map
                  || healthData.iscsi_daemons != null">
-
     <cd-info-card cardTitle="Cluster Status"
                   i18n-cardTitle
                   class="cd-status-card"
                   contentClass="content-highlight"
                   *ngIf="healthData.health?.status">
       <ng-container *ngIf="healthData.health?.checks?.length > 0">
-        <ng-template #healthChecks>
-          <cd-health-checks [healthData]="healthData"></cd-health-checks>
-        </ng-template>
-        <div class="info-card-content-clickable"
-             [ngStyle]="healthData.health.status | healthColor"
-             [ngbPopover]="healthChecks"
-             popoverClass="info-card-popover-cluster-status">
-          {{ healthData.health.status | healthLabel | uppercase }}
-          <i *ngIf="healthData.health?.status !== 'HEALTH_OK'"
-             class="fa fa-exclamation-triangle"></i>
-        </div>
+        <cds-toggletip
+            [dropShadow]="true"
+            placement="bottom">
```

Comment:
```suggestion
           >
```
Same as above, as of latest angualr docs there is no such property.
Also , `autoalign` is set by default, which is good since it adjusts accordingly so we might not want to set any alignment anywhere unless there is some very hard requirement.

Thanks

Reply:
it was required to show on top. As mentioned above